src/masonite/orm/grammar/BaseGrammar.py

```diff
@@ -93,6 +94,8 @@ def _compile_alter(self):
                 if column.length
                 else "",
                 nullable="" if column.is_null else " NOT NULL",
+                after=self.after_column_string().format(after=column.after_column) if column.after_column else ""
```

Comment:
actually just modify this `after=self._compile_column(column.after_column)`

Reply:
That will just have ``name``